### ISSUE TITLE
Deploy broken [remote rejected] asf-site -> asf-site (pre-receive hook declined)

### ISSUE THREAD

**Issue by apupier** [apache/camel-website#821]
```
+ git push --force origin asf-site
remote: Rewinding refs/heads/asf-site is forbidden.
remote:
To https://gitbox.apache.org/repos/asf/camel-website-pub.git
 ! [remote rejected] asf-site -> asf-site (pre-receive hook declined)
error: failed to push some refs to 'https://gitbox.apache.org/repos/asf/camel-website-pub.git'
```

see https://ci-builds.apache.org/job/Camel/job/Camel.website/job/main/820/console

I see no .git/hooks folder (neither a .git folder) in https://gitbox.apache.org/repos/asf?p=camel-website-pub.git;a=tree

**Comment by apupier** [comment]
issue opened upstream https://issues.apache.org/jira/browse/INFRA-23115 (I cannot reopen the issue myself)

**Comment by zregvart** [comment]
Resolved via INFRA-23115, the [#823](https://ci-builds.apache.org/job/Camel/job/Camel.website/job/main/823/) build was successful.
